x-pack/plugins/security_solution/public/management/components/console/components/side_panel/side_panel_flex_item.tsx

```diff
@@ -6,10 +6,18 @@
  */
 
 import React, { memo } from 'react';
-import { EuiFlexItem } from '@elastic/eui';
+import styled from 'styled-components';
+import { EuiFlexItem, transparentize } from '@elastic/eui';
 import { SidePanelContentManager } from './side_panel_content_manager';
 import { useWithSidePanel } from '../../hooks/state_selectors/use_with_side_panel';
 
+const StyledEuiFlexItemWhite = styled(EuiFlexItem)`
+  background-color: ${({ theme: { eui } }) => eui.euiHeaderBackgroundColor} !important;
```

Comment:
Why did you have to use a background color here? 

I think this is ok, but was just curious. We are using a few different Eui properties for background colors (take a look at console.tsx) and should at some time just maybe ensure they are consistent. For this one, just make sure in the dark theme, it looks ok. :)

Reply:
Yeah, it looks good in dark mode